.github/workflows/build.yml

```diff
@@ -220,33 +217,42 @@ jobs:
               generator: "-G'Visual Studio 16 2019' -A x64",
               vcpkg_triplet: x64-windows
             }
-        type: [portable]
+        type: [portable, installer]
 #        type: [portable, installer]
 # The installer type is working, but disabled, to save runner jobs.
 # Enable it, when working on the installer.
 
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v2.4.0
 
       - name: Restore from cache and run vcpkg
-        uses: lukka/run-vcpkg@v5
+        uses: lukka/run-vcpkg@v10.1
         with:
           vcpkgArguments: ${{env.vcpkg_packages}}
           vcpkgDirectory: '${{ github.workspace }}\vcpkg'
           appendedCacheKey: ${{ matrix.config.vcpkg_triplet }}
           vcpkgGitCommitId: ${{ env.VCPKG_VERSION }}
           vcpkgTriplet: ${{ matrix.config.vcpkg_triplet }}
 
-      - name: CMake
+      - name: Checkout IrrlichtMT
+        uses: actions/checkout@v2.4.0
+        with:
+          repository: minetest/irrlicht
+          path: lib/irrlichtmt/
+      - run: |
+            git checkout (git tag --sort=v:refname | Select-Object -Last 1)
```

Comment:
This seems erroneous

Reply:
Fetches the latest tag, not sure if there's a better way yet

Reply:
But you don't want to fetch the latest tag of _Minetest_, you want to run it on the PR commit

Maybe this is supposed to run in the Irrlicht folder? Also, how about uses the `latest` tag on the Irrlicht repo instead?

Reply:
Yeah, Irrlicht https://github.com/minetest/minetest/pull/11176#discussion_r734351505